trackfilter.cc

```diff
@@ -456,7 +456,7 @@ void TrackFilter::trackfilter_split()
           fatal(MYNAME ": invalid time interval unit specified.\n");
         }
 
-        if (TRACKF_DBG) {
+        if constexpr(TRACKF_DBG) {
```

Comment:
I believe "constexpr if" requires the body of the block to be discarded, and your implementation left it up to compiler optimization to do so as it saw fit.  Resharper will flag:

>       trackfilter.cc:23 Variable 'TRACKF_DBG' can be made constexpr.

It will also flag "If statement with constant condition can be replaced with 'if constexpr'', although it didn't find these which is a bit of a mystery.  I have seen that warning in the past.